test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1772,45 +1772,6 @@ static public void testCreateLargeObjectAndMegaRef() throws Throwable {
 		checkFieldAccessMHOfAssortedType(megaGetterAndSetter, megaObjectRef, megaFields, false);
 	}
 
-	/*
-	 * Create a value type and read the fields before
-	 * they are set. The test should Verify that the
-	 * flattenable fields are set to the default values.
-	 * NULL should never be observed for null-restricted fields.
-	 */
-	@Test(priority=4)
-	static public void testDefaultValues() throws Throwable {
-		/* Test with assorted value object with long alignment */
-		MethodHandle makeValueTypeDefaultValueWithLong = lookup.findStatic(assortedValueWithLongAlignmentClass,
-				"makeDefaultValue", MethodType.methodType(assortedValueWithLongAlignmentClass));
-
-		Object assortedValueWithLongAlignment = makeValueTypeDefaultValueWithLong.invoke();
-		for (int i = 0; i < 7; i++) {
-			assertNotNull(assortedValueWithLongAlignmentGetterList[i][0].invoke(assortedValueWithLongAlignment));
-		}
-
-		/* Test with assorted ref object with long alignment */
-		MethodHandle makeRefDefaultValueWithLong = lookup.findStatic(assortedRefWithLongAlignmentClass,
-				"makeDefaultValue", MethodType.methodType(assortedRefWithLongAlignmentClass));
-		Object assortedRefWithLongAlignment = makeRefDefaultValueWithLong.invoke();
-		for (int i = 0; i < 7; i++) {
-			assertNotNull(assortedRefWithLongAlignmentGetterAndSetter[i][0].invoke(assortedRefWithLongAlignment));
-		}
-
-		/* Test with flattened line 2D */
-		MethodHandle makeDefaultValueFlattenedLine2D = lookup.findStatic(flattenedLine2DClass, "makeDefaultValue", MethodType.methodType(flattenedLine2DClass));
-		Object lineObject = makeDefaultValueFlattenedLine2D.invoke();
-		assertNotNull(getFlatSt.invoke(lineObject));
-		assertNotNull(getFlatEn.invoke(lineObject));
-
-		/* Test with triangle 2D */
-		MethodHandle makeDefaultValueTriangle2D = lookup.findStatic(triangle2DClass, "makeDefaultValue", MethodType.methodType(triangle2DClass));
```

Comment:
`makeDefaultValue` and `makeDefaultValueGeneric` can be removed from ValueTypeGenerator.